src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,232 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+// import Duke.Function.*;
+// import Duke.Task.*;
+import java.util.Scanner;
+// import java.util.List;
```

Comment:
It's best to remove any comments that don't serve an explanatory purpose (i.e. they don't elaborate on what the code is doing), including commented-out code.